test/e2e/quick_start_test.go

```diff
@@ -68,3 +68,36 @@ var _ = Describe("Cluster creation with [Ignition] bootstrap [PR-Blocking]", fun
 		})
 	})
 })
+
+var _ = Describe("Cluster Creation using Cluster API quick-start test on vcsim [vcsim]", func() {
```

Comment:
Question: So we explicitly have to set `[vcsim]` here. The spec does not look too different from the normal quickstart?

Note: just want to clarify, I'm ok with this :-)

Reply:
We have a few constraints that make it possible to run vcsim tests together with regular test.
To enforce this, we had two options:
- Manage this manually by adding skip [vcsim] on all the existing tests
- Enforce this programmatically

I went for 2, automatically enforcing skip [vcsim] if there is no explicit focus [vcsim].
As a consequence, we have to have dedicated tests for [vcsim], but I think this is acceptable in this phase where we will have only quick start tests